doc/po4a.7.pod

```diff
@@ -230,6 +230,13 @@ documentation).  Support for this format in po4a is still in its
 infancy.  If you find anything, please file a bug report or feature
 request.
 
+=item simplepod (very highly experimental parser)
```

Comment:
Is it still a very highly experimental parser? I'm just starting this review, but I kinda believe that it's stable now, even if it deserves more tests.